#!/usr/bin/env python
PACKAGE = "vision"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# Params for buoys task
gen.add("loRed", int_t, 0, "Low L bound for Red", 143, 0, 255)
#gen.add("loU", int_t, 0, "Low U bound for Yellow", 17, 0, 255)
gen.add("loGreen", int_t, 0, "Low V bound for Green", 18, 0, 255)
gen.add("hiRed", int_t, 0, "High L bound for Red", 190, 0, 255)
#gen.add("hiU", int_t, 0, "High U bound for Yellow", 48, 0, 255)
gen.add("hiGreen", int_t, 0, "High V bound for Green",49, 0, 255)
gen.add("minContourArea",int_t,0,"Minimum threshold area for buoy contours",194,0,1000)
#gen.add("blur",bool_t,0,"True to enable 5x5 Gaussian blur ",True)



exit(gen.generate(PACKAGE, "flare_task", "flare"))                                                   
